index.bs

```diff
@@ -77,6 +77,30 @@ The module in this new sidebar context will never need to reflow to the wider la
 	<figcaption>A stylesheet heatmap showing added redundancy and rewriting of existing styles required to accommodate the layout variation in <a href="#fig-4">fig. 4</a> </figcaption>
 </figure>
 
+<h2 id="limitations-of-current-techniques">Limitations of current techniques</h2>
+
+There is currently no native CSS mechanism available to developers that achieves the primary function of container queries, which is to <em>employ the full capabilities of CSS to a specific element tree based on the state and properties of their container</em>. Therefore developers must look to workarounds to achieve their goals. These workarounds and their limitations are outlined here:
+
+The use of <strong>Media Queries</strong> can only affect CSS changes based on the state and properties of the viewport. The viewport is not a good indicator of the state and properties of a particular container within the viewport.
```

Comment:
Nit: this is screaming to be a bunch of list items, a definition list (with a brief name for each workaround), or even small sub sections. 

 * complex media queries
 * Flex, Grid, and ???: 
 * iframes

Reply:
We want to highlight how ridiculously complex a media query could be, so showing one here would help.

Reply:
First point - that’s fine, it’s just how I write. I can change that.

Second point - I thought the original work in the Use Cases section inferred that complexity.
The [article by Mat Marquis](https://alistapart.com/article/container-queries-once-more-unto-the-breach) does illustrate it further. Perhaps a link to that? If not, perhaps some else can find one in the wild or write one? I don’t really have the capability to to come up with a good example at the moment and I don’t want to hold things up.
Again, I’d like to avoid bloating this if possible.